test/suites/modules.bash

```diff
@@ -34,38 +36,38 @@ SUITE_modules() {
     # -------------------------------------------------------------------------
     TEST "fall back to real compiler, no sloppiness"
 
-    $CCACHE_COMPILE -x c++ -fmodules -fcxx-modules -c test1.cpp -MD
+    $CCACHE_COMPILE -x c++ -std=c++20 -fmodules -fcxx-modules -c test1.cpp -MD
```

Comment:
Why would you need to force C++20 here? In case you're confusing this for C++20 modules, this is not that, it's Clang modules.

Reply:
I agree, that was a mistake by me.